packages/excalidraw/element/newElement.ts

```diff
@@ -88,6 +89,8 @@ const _newElementBase = <T extends ExcalidrawElement>(
     angle = 0,
     groupIds = [],
     frameId = null,
+    // TODO: think about this more
```

Comment:
This is probably fine for cases such `API.createElement` when we cannot just guess the `fractionalIndex` value.